src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -2356,6 +2356,17 @@ default Emote getEmoteById(long id)
     @Nonnull
     ExplicitContentLevel getExplicitContentLevel();
 
+    /**
+     * Returns the level of multifactor authentication required to execute administrator restricted functions in this guild.
```

Comment:
How is the NSFW level connected to administrator-specific actions?

Reply:
It was copy pasted from the MFALevel. Changed it on last commit.